schemas/objects/classPrefinanceTimelineItem.js

```diff
@@ -0,0 +1,24 @@
+import { prefinanceTimelineItemPreview } from './projectPrefinanceTimelineItem';
+
+export default {
+  title: 'Class Prefinance Timeline Item',
```

Comment:
this is the association of a particular status (eg "Internal review") with some date(s) and current state for a particular credit class timeline